docs/reference/function-configuration/code-entry-types.md

```diff
@@ -0,0 +1,245 @@
+    - `workDir` (dashboard: **Work directory**) (Optional) &mdash; the relative path to the function-code directory within the configured repository branch.
+
+<a id="code-entry-type-github-example"></a>
+#### Example
+
+```yaml
+spec:
+  description: my Go function
+  handler: main:Handler
+  runtime: golang
+  build:
+    codeEntryType: "github"
+    path: "https://github.com/my-organization/my-repository"
+    codeEntryAttributes:
+      branch: "my-branch"
+      headers:
+        Authorization: "my-Github-access-token"
+      workDir: "/go/myfunc"
+```
+
+<a id="code-entry-type-archive"></a>
+### Archive-file code-entry type (`archive`)
+
+Set the [`spec.build.codeEntryType`](/docs/reference/function-configuration.md#spec.build.codeEntryType) function-configuration field to `archive` (dashboard: **Code entry type** = `Archive`) to download a ZIP archive file of the function code either from an [Iguazio Data Science Platform](https://www.iguazio.com) ("platform") data container or from any URL that doesn't require download authentication. The following configuration fields provide additional information for performing the download:
+
+- `spec.build` &mdash;
+  - `path` (dashboard: **URL**) (Required) &mdash; a URL for downloading the archive file.<br/>
+    To download an archive file from an Iguazio Data Science Platform data container, the URL should be set to `<API URL of the platform's web-APIs service>/<container name>/<path to archive file>`, and a respective data-access key must be provided in the `spec.build.codeEntryAttributes.headers.X-V3io-Session-Key` field.
+  - `codeEntryAttributes` &mdash;
+    - `headers.X-V3io-Session-Key` (dashboard: **Access key**) (Required for a platform archive file) &mdash; an Iguazio Data Science Platform access key, which is required when the download URL (`spec.build.path`) refers to an archive file in a platform data container.
+    - `workDir` (dashboard: **Work directory**) (Required) &mdash; the relative path to the function-code directory within the downloaded archive file.
+
+<a id="code-entry-type-archive-example"></a>
+#### Example
+
+```yaml
+spec:
+  description: my Go function
+  handler: main:Handler
+  runtime: golang
+  build:
+    codeEntryType: "archive"
+    path: "https://https://webapi.default-tenant.app.mycluster.iguazio.com/users/myuser/my-functions.zip"
+    codeEntryAttributes:
+      headers:
+        X-V3io-Session-Key: "my-platform-access-key"
+      workDir: "/go/myfunc"
+```
+
+<a id="code-entry-type-s3"></a>
+### AWS S3 code-entry type (`s3`)
+
+Set the [`spec.build.codeEntryType`](/docs/reference/function-configuration.md#spec.build.codeEntryType) function-configuration field to `s3` (dashboard: **Code entry type** = `S3`) to download a ZIP archive file of the function code from an Amazon Simple Storage Service (AWS S3) bucket. The following configuration fields provide additional information for performing the download:
```

Comment:
Same regarding the ZIP archive + formats

Reply:
See the reply to the previous comment.